tap_shopify/schemas/definitions.json

```diff
@@ -417,18 +417,11 @@
         ]
       },
       "accepts_marketing_updated_at": {
-        "anyOf": [
-          {
-            "type": "string" ,
-            "format": "date-time"
-          },
-          {
-            "type": "string"
-          },
-          {
-            "type": "null"
-          }
-        ]
+        "type": [
+          "null",
+          "string"
+        ],
+        "format": "date-time"
```

Comment:
fix date format; all other date format schemas in the schema jsons look like this already